### ISSUE TITLE
Operation not permitted

### ISSUE THREAD

**Issue by victormgomezp** [gitpod-io/gitpod#3953]
### Bug description

Our students cannot enter in GitPod because of this error:

![image](https://user-images.githubusercontent.com/62053368/114877307-716b2280-9dff-11eb-8bf2-a1e62264887b.png)

This happened suddenly, but we dont know how to solve

### Steps to reproduce

just whe you want to open a new workspace

### Expected behavior

enter in the workspace as usual

**Comment by csweichel** [comment]
That most certainly is a bug in gitpod - I'm sorry this is happening.

To narrow the problem down, could you tell me:
- Did they attempt to restart a workspace or create a fresh one from a context URL?
- Also, did they just log in (i.e. went through the log in flow while starting the workspace) or had they logged in before?

**Comment by victormgomezp** [comment]
I am not sure if he restarted it :( 

he was logged in, he didn't log out, but the thing is that some hours after he couldenter in the workspace, but he is not he only one that has claimed this issue.

**Comment by oconnorjoseph** [comment]
@cs

> That most certainly is a bug in gitpod - I'm sorry this is happening.
> 
> To narrow the problem down, could you tell me:
> 
> * Did they attempt to restart a workspace or create a fresh one from a context URL?
> * Also, did they just log in (i.e. went through the log in flow while starting the workspace) or had they logged in before?

This same issue happened to one of my colleagues several hours ago. I started a workspace under a Github organization and was able to access that workspace with no issues. I worked on it for several hours and at one point my connection dropped to the workspace, but after refreshing the page it was restored shortly after (maybe that small detail is useful). I left my instance open and then my college came online several hours later and experienced the above issue. They went through the log in flow while starting the workspace.
